cg/meta/workflow/rnafusion.py

```diff
@@ -278,10 +283,22 @@ def run_analysis(
 
         else:
             LOG.info("Pipeline will be executed using tower")
-            parameters: List[str] = TowerAnalysisAPI.get_tower_launch_parameters(
-                tower_pipeline=self.tower_pipeline,
-                command_args=command_args,
-            )
+            if command_args.resume:
+                from_tower_run_id: int = command_args.id
+                if not from_tower_run_id:
+                    from_tower_run_id: int = TowerAnalysisAPI.get_last_tower_id(
+                        case_id=case_id,
+                        trailblazer_config=self.get_trailblazer_config_path(case_id=case_id),
+                    )
+                LOG.info(f"Pipeline will be resumed from run {from_tower_run_id}.")
+                parameters: List[str] = TowerAnalysisAPI.get_tower_relaunch_parameters(
+                    from_tower_id=from_tower_run_id, command_args=command_args.dict()
+                )
```

Comment:
The function is quite long. It would be nice to refactor this section in a general Tower function.

Reply:
Agree. It's quite ugly. I'll refactor this as part of https://github.com/Clinical-Genomics/cg/pull/2156